dspace/solr/search/conf/schema.xml

```diff
@@ -240,7 +240,7 @@
     <field name="search.resourceid" type="string" indexed="true" stored="true" required="true" omitNorms="true" />
 
     <!-- Internal DSpaceObject Type ID -->
-    <field name="search.resourcetype" type="sint" indexed="true" stored="true" required="true" omitNorms="true" />
+    <field name="search.resourcetype" type="string" indexed="true" stored="true" required="true" omitNorms="true" />
```

Comment:
The type `string` is tokenized, this should rather be the type `keywordFilter`

Reply:
This shouldn't be an issue, the string type is a simple type that doesn't do any alterations to it, see: https://github.com/DSpace/DSpace/blob/4c9a30bb14e2a6926ecef89efcb8f08721fafce2/dspace/solr/search/conf/schema.xml#L31. This string is also used for search.resourceid for example.